### DISCUSSION TITLE
Gitter was down (back now)

### DISCUSSION THREAD

**Discussion by vytas7** [falconry/falcon#1998] [General]
It looks like Gitter is down at the time of this writing (Wed Dec 22 19:39:13 CET 2021)
![image](https://user-images.githubusercontent.com/3430939/147139996-5a551a82-b2be-4baf-b981-c249ee3a6bcb.png)

We can use Discussions in the case anyone has a pressing issue :thinking:

**Comment by CaselIT** [comment]
Seems to be connected to an aws outage

**Reply by vytas7** [reply]
That's a shame, still down... Somehow Discussions don't feel like the same realtime chat :grimacing:

**Reply by vytas7** [reply]
(Feels still more like an issue tracker.)

**Reply by vytas7** [reply]
And indeed it does @CaselIT 
https://www.theverge.com/2021/12/22/22849780/amazon-aws-is-down-outage-slack-imgur-hulu-asana-epic

**Reply by CaselIT** [reply]
I usually find out by browsing hacker news https://news.ycombinator.com/item?id=29648286
